reports-scheduler/src/main/kotlin/org/opensearch/reportsscheduler/model/ReportDefinition.kt

```diff
@@ -80,7 +80,7 @@ import java.time.Duration
  *       "title":"title",
  *       "textDescription":"textDescription",
  *       "htmlDescription":"optional htmlDescription",
- *       "channelIds":["optional_channelIds"]
+ *       "configIds":["optional_configIds"]
```

Comment:
could you also mention in PR description that you changed `channelIds` to `configIds` in reports scheduler

Reply:
Updated, specified reports-scheduler vs. just backend